kymatio/scattering1d/filter_bank.py

```diff
@@ -722,10 +734,10 @@ def scattering_filter_factory(J_support, J_scattering, Q, r_psi=math.sqrt(0.5),
         psi2_f[n2]['j'] = j2
     phi_f['xi'] = 0.
     phi_f['sigma'] = sigma_low
-    phi_f['j'] = 0
+    phi_f['j'] = log2_T
```

Comment:
Why?

Reply:
In fact this breaks because T is in the function signature and log2_T is not defined here

Reply:
@pawarrick I don't follow, tests pass for me. `log2_T` is always defined:

```python
    log2_T = math.floor(math.log2(T))
    if max_subsampling is None:
```

Reply:
@janden Because `0` is wrong?

Reply:
@pawarrick Nevermind you moved it - good catch.